Shared/Samples/Densify and generalize geometry/DensifyAndGeneralizeGeometryView.SettingsView.swift

```diff
@@ -0,0 +1,69 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+extension DensifyAndGeneralizeGeometryView {
+    struct OptionsView: View {
+        /// The view model for the sample.
+        @ObservedObject var model: Model
+        
+        var body: some View {
+            List {
+                // Generalize toggle and slider.
+                Section {
+                    Toggle("Generalize", isOn: $model.shouldGeneralize)
+                        . onChange(of: model.shouldGeneralize) { _ in
+                            model.updateGraphics()
+                        }
+                    VStack {
+                        HStack {
+                            Text("Max Deviation")
+                            Spacer()
+                            Text(String(Int(model.maxDeviation)))
+                        }
+                        Slider(value: $model.maxDeviation, in: 1...250) { _ in
+                            model.updateGraphics()
+                        }
```

Comment:
The slider's [`onEditingChanged`](https://developer.apple.com/documentation/swiftui/slider) closure doesn't match the old behavior. The `onEditingChanged` closure is only called twice: touch down (true) and touch up (false) on the knob. Whereas the old one changes constantly when the values are updated.

I'll leave it open for the other reviewer's thoughts. I'd like to have the old behavior but the new one isn't a big problem.